src/main/java/mil/dds/anet/beans/Task.java

```diff
@@ -97,30 +97,30 @@ public void setCategory(String category) {
 		this.category = Utils.trimStringReturnNull(category);
 	}
 
-	@GraphQLFetcher("parentTask")
-	public Task loadParentTask() {
-		if (parentTask == null || parentTask.getLoadLevel() == null) { return parentTask; }
-		if (parentTask.getLoadLevel().contains(LoadLevel.PROPERTIES) == false) { 
-			this.parentTask = AnetObjectEngine.getInstance()
-					.getTaskDao().getById(parentTask.getId());
+	@GraphQLFetcher("customFieldRef1")
+	public Task loadCustomFieldRef1() {
+		if (customFieldRef1 == null || customFieldRef1.getLoadLevel() == null) { return customFieldRef1; }
+		if (customFieldRef1.getLoadLevel().contains(LoadLevel.PROPERTIES) == false) {
+			this.customFieldRef1 = AnetObjectEngine.getInstance()
+					.getTaskDao().getById(customFieldRef1.getId());
 		}
-		return parentTask;
+		return customFieldRef1;
 	}
 
-	public void setParentTask(Task parent) {
-		this.parentTask = parent;
+	public void setCustomFieldRef1(Task customFieldRef1) {
+		this.customFieldRef1 = customFieldRef1;
 	}
 
 	@GraphQLIgnore
-	public Task getParentTask() { 
-		return this.parentTask;
+	public Task getCustomFieldRef1() {
+		return this.customFieldRef1;
 	}
 
 	@GraphQLFetcher("childrenTasks")
```

Comment:
Should these still be called `childrenTasks` now that the inverse is no longer called `parentTask`?

Reply:
Or better yet: is `childrenTasks` actually used **at all** outside the server-side tests? If not, we might as well drop it altogether.